.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -113,7 +113,7 @@ runs:
     uses: docker/metadata-action@e5622373a38e60fb6d795a4421e56882f2d7a681 # v3.6.2
     with:
       flavor: |
-        suffix=-root
```

Comment:
Do we want this here?

```
latest=false
```

Reply:
This will publish `latest-root` instead, dont think we should remove it, should we?

Reply:
Nah, let's keep it then 👍